update_ip/ip_getters/getters.py

```diff
@@ -1,24 +1,45 @@
-from update_ip.ip_getters.base import BaseIpGetter,  get_ip_from_http
+from update_ip.ip_getters.base import BaseIpGetter
 
 
 class DynDns(BaseIpGetter):
-    NAME= "dyndns.org"
-    URL= "http://checkip.dyndns.org"
+    NAME = "dyndns.org"
+    URL = "http://checkip.dyndns.org"
 
 
-'''
-class IpChicken(BaseIpGetter):
-    NAME= "ipchicken.com"
-    URL= "http://ipchicken.com/"
-'''
-
 class IfconfigMe(BaseIpGetter):
-    NAME= "ifconfig.me"
-    URL= "http://ifconfig.me/ip"
+    NAME = "ifconfig.me"
+    URL = "http://ifconfig.me/ip"
+
+
+class IPEchoNet(BaseIpGetter):
+    NAME = "ipecho.net"
+    URL = "http://ipecho.net/plain"
+
+
+class IP4Me(BaseIpGetter):
+    NAME = "ip4.me"
+    URL = "http://ip4.me/"
+
+
+class WhatIsMyPublicIP(BaseIpGetter):
+    NAME = "whatismypublicip.com"
+    URL = "http://www.whatismypublicip.com/"
+
+
+class WhatIsMyIP(BaseIpGetter):
```

Comment:
I've removed ifconfig.me and whatismyip.com because fails in test.

A solution will be change headers to look like a browser and also clean all html tags and decode al html chars but i will better do this in another PR